nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -36,6 +37,52 @@
 namespace nebula::drivers
 {
 
+struct HesaiDecodeFilteredInfo
+{
+  uint16_t distance_counter = 0;
+  uint16_t fov_counter = 0;
+  uint16_t timestamp_counter = 0;
```

Comment:
Please add `filtered` to the names, e.g. `n_filtered_distance`, `n_filtered_fov` etc.